### PR TITLE
Add @earth_age_10m to the CI cache

### PR DESCRIPTION
**Pull request by meghanrjones**
**Description of proposed changes**

`test/grdimage/twogrids.sh` was failing at https://github.com/GenericMappingTools/gmt/runs/4242161710?check_suite_focus=true due to a curl timeout. This PR caches @earth_age_10m to resolve the issue.

Fixes #

**Reminders**

- [ ] Make sure that your code follows our style. Use the other functions/files as a basis.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Describe changes to function behavior and arguments in a comment below the function declaration.
- [ ] If adding new functionality, add a detailed description to the documentation and/or an example.

### CODE REVIEW

**Review by PaulWessel — APPROVED** [review]

**Comment by meghanrjones** [comment]
Just a note to not merge this yet, since I need to recomment the trigger on pull request line before merging.

**Pull request merged**
